### ISSUE TITLE
Could use a basic guide for installation and development

### ISSUE THREAD

**Issue by PeterDavidCarter** [ArangoDB-Community/pyArango#167]
At the moment the readme and the docs are very much geared towards people wanting to develop on a fork, rather than wanting to develop something else using pyArango. Could use a quickstart section dealing with installation via pip, adding connection string etc. Am happy to write this if it's desirable.

**Comment by NetwarSystem** [comment]
I agree on this - I've been looking for an example of inserting a JSON object using pyArrango ... nothing pops to the surface with Google, Stackoverflow's newest dates to 2017, etc.
